test/Handler/SitesSpec.hs

```diff
@@ -108,3 +111,9 @@ spec = withApp $ do
 
 createSite_ :: UserId -> Site -> DB ()
 createSite_ userId = void . createSite userId
+
+selectLanguage :: Text -> RequestBuilder m ()
+selectLanguage lang = do
```

Comment:
> Is there no general helper to select an option by label or value?

Right. This would be great to add, but I don't know if its existence yet.

> We have to select by index?

Or we could write something that does CSS queries to find the `select` by label (after reproducing Yesod.Test's internal `nameFromLabel` function) then find within it an the correct `option`, then using that `option`'s value. This just seemed easier since `langList` is already available and Yesod creates indexed option-values.